lib/http.js

```diff
@@ -158,12 +174,37 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         accountId: account
       });
       if(profile) {
+        const {meters} = await profileMeters.findByProfile({
+          profileId: profile
+        });
+        // Note: In the next major release, this API should not repeat the same
+        //       set of meters for each profile.
         const records = profileAgentRecords.filter(({profileAgent}) => {
           return profileAgent.profile === profile;
-        });
+        }).map(r => ({...r, meters}));
         return res.json(records);
       }
-      res.json(profileAgentRecords);
+
+      const profileMetersMap = new Map();
+      const promises = profileAgentRecords.map(async record => {
+        const {profile} = record.profileAgent;
+        let promise = profileMetersMap.get(profile);
+        if(!promise) {
+          promise = profileMeters.findByProfile({profileId: profile});
+          profileMetersMap.set(profile, promise);
+        }
+
+        const {meters} = await promise;
+        return {
+          ...record,
+          meters
```

Comment:
Should these meters be mixed into the top-level of the record or into the profile agent within the record?

Reply:
I intentionally left it out of the record because they are not a property of profile agents.